wbooks/src/app/components/HeaderBackground/styles.ts

```diff
@@ -0,0 +1,7 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  image: {
+    width: '100%'
```

Comment:
Missing `flex` (I think, test if it's necessary or not) and `backgroudColor`

Reply:
If I added `flex: 1`, this convert the `height` to `90`, without `flex: 1`, the image `height` is `102`